pynest/nest/lib/hl_api_info.py

```diff
@@ -241,6 +241,9 @@ def GetStatus(nodes, keys=None):
         String or a list of strings naming model properties. GetDefaults then
         returns a single value or a list of values belonging to the keys
         given.
+    output: str, optional
+        Whether the returned data should be in a JSON serializable format
+        (output='json'). Default is ''.
```

Comment:
Again, `output='json'` suggests that the output already _is_ JSON. If I were to use this argument, I would really expect the function to return a JSON string instead of still a Python dictionary with all iterables converted to plain lists.

The same applies to `GetDefaults` below.